### PR TITLE
String updates for MPP-2970

### PR DESCRIPTION
**Pull request by lloan**
String updates:

hero-section-social-proof = As seen in

For Relay PR: https://github.com/mozilla/fx-private-relay/pull/3295/

### CODE REVIEW

File: en/landing.ftl [hunk 1]
```diff
@@ -9,6 +9,7 @@ hero-section-body = Our secure, easy-to-use email and phone masks help keep your
     private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and 
     get only the emails you want in your inbox.
 hero-section-cta = Get started
+hero-section-social-proof = As seen in
```
Comment (peiying2) [inline en/landing.ftl:12]: What is followed after this?

Comment (lloan) [reply en/landing.ftl:12]: @peiying2 sorry, here is the context: 

image

Comment (peiying2) [reply en/landing.ftl:12]: Thank you for the screenshot. Please add a comment by saying something like "website magazines such as xyz, etc". We want the localizers to be able to see it.

**Comment by maxxcrawford** [comment]
Closing because of the larger FTL changes from @codemist. Will reopen with new layout.

**Pull request closed without merging**